CONTRIBUTING.md

```diff
@@ -0,0 +1,487 @@
+    * UI themes are named `[theme-name]-ui`
+    * Syntax themes are named `[theme-name]-syntax`
+    * Often themes that are designed to work together are given the same root name, for example: `one-dark-ui` and `one-dark-syntax`
+    * UI themes style everything outside of the editor pane &mdash; all of the green areas in the [packages image above](#pulsar-packages-image)
+    * Syntax themes style just the items inside the editor pane, mostly syntax highlighting
+* Packages that add [autocomplete providers](https://github.com/pulsar-edit/autocomplete-plus/wiki/Autocomplete-Providers) are named `autocomplete-[what-they-autocomplete]` &mdash; ex: [autocomplete-css](https://github.com/pulsar-edit/autocomplete-css)
+
+### Design Decisions
+
+When we make a significant decision in how we maintain the project and what we can or cannot support, we will document it in the [pulsar-edit/design-decisions repository](https://github.com/pulsar-edit/design-decisions). If you have a question around how we do things, check to see if it is documented there. If it is *not* documented there, please open a new topic on [Github Discussions, the official Pulsar message board](https://github.com/orgs/pulsar-edit/discussions) and ask your question.
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Pulsar. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out [the required template](https://github.com/pulsar-edit/.github/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+
+* **Check the [debugging guide](https://flight-manual.atom.io/hacking-atom/sections/debugging/).** You might be able to find the cause of the problem and fix things yourself. Most importantly, check if you can reproduce the problem [in the latest version of Pulsar](https://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version), if the problem happens when you run Pulsar in [safe mode](https://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode), and if you can get the desired behavior by changing [Atom's or packages' config settings](https://flight-manual.atom.io/hacking-atom/sections/debugging/#check-atom-and-package-settings).
```

Comment:
These links to the flight manual can probably be temporarily linked to their alternative under the docs repo until we're able to get that up.

Reply:
Looking at the docs repo, should we wait till the GitHub Pages is functional, or do you think its worthwhile to have it just point to the raw MarkDown on GitHub